### PR TITLE
50p ui

### PR DESCRIPTION
**Pull request by StephanieBr**
screencapture-localhost-4000-2018-1515653089760 1
**Event title:**

**Organizer's name:**

cc @hasgeek/events-review

### CODE REVIEW

**Comment by StephanieBr** [comment]
screencapture-localhost-4000-2018-1516606513233

File: tests/before_build/steps/events.py [hunk 1]
```diff
@@ -144,6 +144,24 @@ def valid_external_url(field, value, error):
...
 }
 
+banner_schema = {
+    'image_url': {
+        'validator': valid_external_url,
+        'required': False
+    }
+}
+
+featured_video_schema = {
```
Comment (akshaybhalotia) [inline tests/before_build/steps/events.py]: Change this to `youtube_url_schema`

Comment (StephanieBr) [reply tests/before_build/steps/events.py]: okay, changed. Waiting for checks to pass

File: tests/before_build/steps/events.py [hunk 2]
```diff
@@ -231,6 +249,31 @@ def valid_external_url(field, value, error):
...
+        },
+        'required': False
+    },
+    'youtube_url': {
+        'type': 'dict',
+        'schema': {
+            'title': {
+                'type': 'string',
+                'required': False
+            }
```
Comment (akshaybhalotia) [inline tests/before_build/steps/events.py]: Add a `,` after `}`

File: _50p_events/2018.md [hunk 3]
```diff
@@ -14,8 +14,10 @@ description: "50p is India's first independently curated conference on payments.
 If you are working on any of the above areas, you should come to 50p."
 boxoffice_item_collection: '685e9db7-b62d-4137-895f-f734dc216c4d'
 logo:
-  image_url: /images/50p-logo.png
+  image_url: https://images.hasgeek.com/embed/file/a149190333084b5d80393c19104a878e
   has_title: true
+banner:
```
Comment (akshaybhalotia) [inline _50p_events/2018.md]: Any specific reason `banner` contains an `image_url`? Can it be put directly without nesting?

**Pull request merged**